packages/api-specification/interface/screen.ts

```diff
@@ -0,0 +1,36 @@
+declare namespace ssf {
+  export abstract class Screen {
+    static getDisplays(): Promise<Array<Display>>;
+  }
+
+  /**
+   * Information about the users display. Note that the Browser API does not support multiple
+   * displays, so it assumes the display the browser is running in is the only display.
+   */
+  export interface Display {
+    /**
+     * Unique Id of the display.
+     */
+    id: string;
```

Comment:
Ok. If everyone is happy I will merge his as is and add an issue to add this functionality (along with updating the comment once the functionality is there)?